src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -164,6 +164,7 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 	private currSelectedItem: number = 0;
 	private hasSeperator: boolean = false;
 	private block?: HTMLElement;
+	private allowEnterClick: boolean = true;
```

Comment:
Nit: `Enter` is contributed via keybinding/command, it would be better if we set context (e.g., `code-action-enabled`) on list items and enforce the context on keybindings directly.